tests/test_env_parsing.py

```diff
@@ -0,0 +1,15 @@
+from classy_config import ConfigValue
+from classy_config.config import get_raw_config, register_config
+
+test_values = [
+    "1",
+    "2",
+    "3",
+    "4"
+]
+
+def test_env_config():
+    register_config(filepath="tests/configs/test-config.env", prefix="env")
```

Comment:
Could you please also add a test for using system environment variables? 

I would expect doing
```py
import os

os.environ["test_5"] = 5
```
before registering the config to work. 

Althought there maybe a cleaner way to do this using `conftest.py` or even set within the `pyproject.toml`